sdk/samples/iot/iot_sample_common.c

```diff
@@ -491,13 +489,14 @@ void iot_sample_generate_sas_base64_encoded_signed_signature(
 {
   IOT_SAMPLE_PRECONDITION_NOT_NULL(out_sas_base64_encoded_signed_signature);
 
-  int rc;
+  az_result rc;

```

Comment:
humm `rc` is return code...but we are not returning it. exit_code?

Reply:
this is all being removed in my next PR